image/Image/Image.css

```diff
@@ -0,0 +1,34 @@
+@import "../../utility/component.css";
+
+/*image size*/
+
+.image-size-extrasmall{
+    width: 1.25rem;
+    height: 1.25rem;
+}
+
+.image-size-small{
+    width: 3.125rem;
+    height: 3.125rem;
+}
+
+.image-size-medium{
+    width: 6.25rem;
+    height: 6.25rem;
+}
+
+.image-size-larger{
+    width: 12.5rem;
+    height: 12.5rem;
+}
+
+/*image responsive restricted upto 25rem*/
+.image-responsive{
+    width: 100%;
+    max-width: 25rem;
```

Comment:
why the image was restricted to max-width: 25 rem,
Also, you can add object-fit: contain/cover to make it more precise.